package-lock.json

```diff
@@ -294,7 +294,7 @@
     "anymatch": {
       "version": "1.3.2",
       "resolved": "https://registry.npmjs.org/anymatch/-/anymatch-1.3.2.tgz",
-      "integrity": "sha512-0XNayC8lTHQ2OI8aljNCN3sSx6hsr/1+rlcDAotXJR7C1oZZHCNsfpbKwMjRA3Uqb5tF1Rae2oloTr4xpq+WjA==",
+      "integrity": "sha1-VT3Lj5HjyImEXf26NMd3IbkLnXo=",
```

Comment:
How did these changes to `package-lock` get here? Looks like just a different hash scheme but otherwise presumably equivalent... how can we avoid this? (cc @etpinard )

Reply:
Hmm that's strange. @monfera which node and npm versions are you using?

Reply:
@alexcjohnson  @etpinard I'm on npm 5.6.0 and node 9.3.0 but had no intention of checking in the `package-lock.json`, removed before the merge.

Reply:
Interesting. Maybe node 9.x uses different hashing scheme.

Reply:
Looks like various things may cause a flip to sha1, discussed eg. [here](https://github.com/npm/npm/issues/16938#issuecomment-339863980)